### ISSUE TITLE
DNS records not updated when renaming a VM

### ISSUE THREAD

**Issue by oakey-b1** [sapcc/elektra#341]
If renaming a VM the `dns_name` and `dns_assignment` fields of the port(s) are not updated.
Besides that there doesn't seam to be a way to update these records over the dashboard.

**Comment by Carthaca** [comment]
I agree, elektra would benefit from this. Again a customer was confused about that behaviour and missing means to change the DNS name.

- [ ] port show does not show the fields mentioned above
- [ ] port update does not allow to change the dns_name
- [ ] server rename could be improved by an optional flag to also change the dns_name, so our customers can decide if the dns_name of the server port should be updated, too or not
